api/prisma/schema.prisma

```diff
@@ -14,16 +14,30 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum Roles {
+  USER
+  VOLONTEER
+}
+
```

Comment:
Remove repeated enums and import them from utils folder

Reply:
Thanks for the comments!😊 But in the file with the ".prisma" extension, I can't do a typical enam import from another file, so I did it directly in this file. And the enums in the "enums" folder are created for further use in services. I'll think about how to fix the code duplication.